src/snowflake/snowpark/_internal/proto/ast_pb2.py

```diff
@@ -11,582 +11,589 @@
 _sym_db = _symbol_database.Default()
 
```

Comment:
has this been updated with the most recent one from the server / mono repo branch?

Reply:
Yes it is the most recent one